book/src/specs/kimchi.md

```diff
@@ -1376,83 +1376,128 @@ You can find these operations under the [proof creation](#proof-creation) and [p
 A proof consists of the following data structures:
 
 ```rs
-#[derive(Clone)]
-pub struct LookupEvaluations<Field> {
+#[serde_as]
+#[derive(Clone, Deserialize, Serialize)]
+pub struct LookupEvaluations<F: CanonicalSerialize + CanonicalDeserialize> {
```

Comment:
might be nicer to just have `F: Field` in the trait bound (this is what we do everywhere else)

Reply:
I was thinking the same, but maybe it's also good the way Anais has it (to keep the struct as general as possible and then put the `Field` bound on the functions in the impl)?